opm/autodiff/SimulatorFullyImplicitBlackoil_impl.hpp

```diff
@@ -102,10 +102,12 @@ namespace Opm
         const parameter::ParameterGroup param_;
 
         // Parameters for output.
-        bool output_;
-        bool output_vtk_;
-        std::string output_dir_;
-        int output_interval_;
+        const bool output_;
```

Comment:
This `const` change (and its following changes below) means that even if `output=false` the parameters `output_vtk` etc. will still be asked for which is slightly less bad. Is there a reason why you made this change?